### ISSUE TITLE
Extended API: Open Chat

### ISSUE THREAD

**Issue by JosephP91** [finos/symphony-bdk-java#477]
## Extended API: Open Chat

Hello everyone! I was wondering if I can open a specific Symphony chat using the Extended Javascript API. Is it possible?

**Comment by symphony-youri** [comment]
Hello @JosephP91,

In 20.10 new extension APIs have been added to open chats, please check https://docs.developers.symphony.com/v/v20.10/building-extension-applications-on-symphony/overview-of-extension-api/extension-api-services/ui-service (openIMbyStreamID and openIMbyUserIDs).

**Comment by JosephP91** [comment]
Hello @symphony-youri 

thanks, I'll check the link

**Comment by JosephP91** [comment]
@symphony-youri I just checked out the link. Inside the uiService object I can see only these method:

`["start", "registerExtension", "unregisterExtension", "getExtensions", "triggerExtension", "systemMessage","trigger", "filter"]`

I suppose I am not using the 20.10 version. Is there a way to select the version to use? Currently I am using this link to use the JS API: https://cdn.symphony.com/resources/api/v1.0/symphony-api.js

**Comment by symphony-youri** [comment]
You can still give it a shot even the method does not appear in the published symphony-api.js (we are in the process of updating it). If the Symphony client is at least 20.10 it should work.

**Comment by JosephP91** [comment]
@symphony-youri actually I am using the web browser and not the Symphony desktop client. Does this has something to do with the POD version?

**Comment by symphony-adnane** [comment]
Hey @JosephP91 even if the client is web it should work normally. 
Do you have access to Client 2.0 ?

**Comment by JosephP91** [comment]
Hi @symphony-adnane! I was confusing with the POD version. I just switched to Symphony 2.0 (Web version) and it works. One question, this API will not work on the Symphony 1.5 client, right?

**Comment by symphony-adnane** [comment]
@JosephP91 it should normally work if you're on version 20.10+ in 1.5 
so maybe an update is needed for your POD 
The difference with 2.0 is that you'll always have access to the latest client version 🚀  :)

**Comment by JosephP91** [comment]
I can confirm that it's working in the 2.0 web client. In the 1.5 version, it's not working

**Comment by symphony-youri** [comment]
You should be able to check the client 1.5 version with https://XXX.symphony.com/client/version.json

**Comment by JosephP91** [comment]
Using that link, I get the following response: 

`{
version: "43.0.0-10917",
builtOn: "Tue Jul 21 03:01:11 UTC 2020",
forceUpgrade: true,
branch: "20.5.1",
gitHash: "7e80a0f473e643685f2654bb1ee9e8b8a29a90f7",
}`

**Comment by symphony-youri** [comment]
Indeed, so until you receive a 20.10+ version, this feature will only be available on client 2.0.
